internal/commands/resolve.go

```diff
@@ -86,8 +117,31 @@ func ResolveIncidentByDialog(ctx context.Context, client bot.Client, logger log.
 		return err
 	}
 
-	channelAttachment := createResolveChannelAttachment(incident, userName)
-	privateAttachment := createResolvePrivateAttachment(incident)
+	isPostMortemMeeting, err := strconv.ParseBool(postMortemMeeting)
+	if err != nil {
+		return err
+	}
+
+	if isPostMortemMeeting {
```

Comment:
What do you think about remove this condition to a private method?

Reply:
I dont understood this propose :thinking:

Reply:
I think everything inside this if could be inside a private method but is just a provocation, it's not necessary.